webapp/src/DOMJudgeBundle/Twig/TwigExtension.php

```diff
@@ -893,4 +894,32 @@ public function wrapUnquoted(string $text, int $width = 75, string $quote = '>')
 
         return $result;
     }
+
+    /**
+     * Convert a hex color to RGBA
+     * @param string $text
+     * @param float  $opacity
+     * @return string
+     */
+    public function hexColorToRGBA(string $text, float $opacity = 1): string
+    {
+        $col = Utils::convertToHex($text);
+        preg_match_all("/[0-9A-Fa-f]{2}/", $col,$m);
+        if (!count($m)) {
+            return $text;
+        }
+
+        $m = current($m);
+        switch (count($m)){
```

Comment:
Some formatting here :)

Reply:
Resolved in f7ccd52cd87e43d97000bd0aca80af5607a28160